sc.it.homework.Concurrency/src/main/java/Concurrency/FestivalGate.java

```diff
@@ -0,0 +1,32 @@
+package Concurrency;
+import java.util.LinkedList;
+public class FestivalGate {
+    public static void main(String[] args) {
+        generateStatistics();
+    }
+    private static void generateStatistics() {
+        FestivalStatisticsThread fst = new FestivalStatisticsThread("Statistics");
+        Thread newTrd = new Thread(fst);
+        newTrd.start();
+        try{
+            Thread.sleep(5000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+    static synchronized  LinkedList<String> generateAttendeeThread() {
+        LinkedList<String> list = new LinkedList<>();
+        for (int i = 0; i < 100; i++) {

```

Comment:
is it not unlimited, you could use a huge random number or while(true) / infinite loop